Lib9c/BlockHelper.cs

```diff
@@ -38,9 +37,12 @@ public static class BlockHelper
             var redeemCodeListSheet = new RedeemCodeListSheet();
             redeemCodeListSheet.Set(tableSheets[nameof(RedeemCodeListSheet)]);
 
-            // FIXME Must use a separate key for the mainnet.
-            var minterKey = new PrivateKey();
-            var ncg = new Currency("NCG", 2, minterKey.ToAddress());
+            if (privateKey is null)
+            {
+                privateKey = new PrivateKey();
```

Comment:
When the `--private-key` option is omitted, how about showing the password prompt through tty rather than creating a random one?

Reply:
🆗 I'll try

Reply:
Instead of password prompt, I turned `--private-key` option to required. (IMHO it also help prevent mistake)